lib/network/probe/ProbeServer.js

```diff
@@ -4,7 +4,7 @@ const errors = require('../../errors');
 
 const DEFAULT_LIVE_ROUTE = '/_/live';
 const DEFAULT_READY_ROUTE = '/_/ready';
```

Comment:
I think while we are here we can change liveness and ready paths as well

Reply:
This is not in the scope of this PR.
Let's focus on the metrics route for now.